contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,29 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of problematic language from the documentation, use the following terminology when referring to OpenShift control plane or compute nodes:
```

Comment:
Maybe also add something in this initial sentence identifying "master" (and "worker", if appropriate) as the problematic language

Reply:
I have now updated this line to the following:

"To assist with the removal of the problematic word "master" from the documentation, use the following terminology when referring to OpenShift control plane nodes:"